app/uk/gov/hmrc/cdsimportsddsfrontend/controllers/forms/DocumentationTypeFormMapping.scala

```diff
@@ -28,14 +28,21 @@ object DocumentationTypeFormMapping {
     "previousDocGoodsItemId" -> optional(text),
     "additionalInfoCode" -> optional(text),
     "additionalInfoDescription" -> optional(text),
-    "additionalDocCategoryCode" -> optional(text),
-    "additionalDocTypeCode" -> optional(text),
-    "additionalDocId" -> optional(text),
-    "additionalDocLPCO" -> optional(text),
-    "additionalDocName" -> optional(text),
+    "additionalDocument" -> seq(mapping(
```

Comment:
could these mappings be extracted to separate files?

Reply:
+1